src/app/story/storyrow/totaltimes/totaltimes.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="mobile-hide">
+    <table class="timeline-properties">
+        <tbody>
+          <tr class="timeline-properties__row">
+            <td class="timeline-data">Total lead time: </td>
```

Comment:
Could you please remove the space after the colon? I don't think it does anything. Could you also do this for the others please?